sotopia/agents/llm_agent.py

```diff
@@ -135,6 +135,7 @@ async def aact(self, obs: Observation) -> AgentAction:
         history = "\n".join(
             f"{y.to_natural_language()}" for y in message_to_compose
         )
+        # The print behavior is weird in async mode
```

Comment:
Agree. Maybe we could comment these two lines as they are in fact not so helpful in debugging?

Reply:
good point! i'll remove them for now.